changes/test_tasks.py

```diff
@@ -80,6 +83,35 @@ def create_outbound_lookup(self, result_count=1):
             match_querystring=True,
         )
 
+    def create_identity_lookup_with_id_field(self, lang="eng_ZA"):
+        responses.add(
+            responses.GET,
+            "http://is/api/v1/identities/test-identity-uuid/",
+            json={
+                "results": [
+                    {"id": "test_identity-uuid", "details": {"lang_code": lang}}
+                ]
+            },
```

Comment:
I am unable to create a mock assert ```{
                "test_identity-uuid",
                "details": {"lang_code": "eng_ZA", "timeout_timestamp": timestamp},
            }```

I get a python error afterwards, how do I achieve this?

Reply:
I think then in your code, there's something that's expecting a `results` key with a list of identities. It's then correct that the test is failing, because the actual API won't return that, so if we ran it on the real identity store, we would get the same python error. So we should fix the code that's processing the result of the call to the identity store